EST.MIT.InvoiceImporter.Function/Functions/ImporterFunctions.cs

```diff
@@ -57,4 +70,23 @@ public ImporterFunctions(IAzureBlobService azureBlobService, IAzureTableService
             throw;
         }
     }
+
+    private Notification CreateNotificationRequest(ImportRequest importMessage)
+    {
+        var baseUrl = _configuration.GetValue<string>("WebUIBaseUrl");
+
+        return new NotificationBuilder()
+                                        .WithId(importMessage.ImportRequestId.ToString())
+                                        .WithScheme(importMessage.SchemeType)
+                                        .WithAction(importMessage.Status)
+                                        .WithEmailRecipient(importMessage.Email)
+                                        .WithData(new NotificationOutstandingApproval
+                                        {
+                                            Name = importMessage.Email,
+                                            Link = $"{baseUrl}/user-uploads",
```

Comment:
the "/user-uplloads" bit also needs to be in config. It shouldn't be hard coded.